script/Arcueid/Day 4/04_00_ARC04_3.txt

```diff
@@ -86,12 +92,13 @@ Everyone is taking it easy, probably because there is plenty of time before home
 [sha:37988a64e9c4a96d7fb21b1bd74cc7f505197a95]{
 -- Page 107, Offset 5827.
 -- もちろん自分もその一人だ。
-Of course, I'm one of them.
+Of course, as I am I.
```

Comment:
Typo? Might also just go for a more casual
> Of course, that includes me.

Reply:
Typo